test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const Access = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const query = {
+  description: 'Example of a gist',
+  files: {
+    'README.md': {
+      content: 'This is a gist example'
+    }
+  }
+};
+describe('DELETE Consumption and resources doesn not exist', () => {
+  let createGist;
+  before(async () => {
+    createGist = await Access.post(`${urlBase}/gists`, query);
+  });
+  it('Create a gist with POST and Promises', () => {
```

Comment:
I suggest changing the name because inside it you're not creating the gist but you're verifying the gist created in the before

Reply:
yeah, something like 'Verify creation of gist used for tests'